pkg/controller/machine/controller_test.go

```diff
@@ -94,3 +90,27 @@ func machineControllerReconcile(t *testing.T, cs *clientset.Clientset, controlle
 		t.Fatalf("reconcile never finished")
 	}
 }
+
+func machineControllerConcurrentReconcile(t *testing.T, cs *clientset.Clientset, ta *TestActuator) {
+
+	// Enable Delay completion feature.
+	ta.DelayInSeconds = 1
+
+	client := cs.ClusterV1alpha1().Machines("default")
+	// Create an instance
+	for i := 0; i < 10; i++ {
+		instance := v1alpha1.Machine{}
+		instance.Name = "instance" + strconv.Itoa(i)
+		if _, err := client.Create(&instance); err != nil {
+			t.Fatal(err)
+		}
+	}
+
+	time.Sleep(2 * time.Second)
+
+	// Sometime the reconcilation will trigger the idempotent action which will not call into
+	// actuator. 3 (larger than 2) is good enough to verify parallel execution.
+	if ta.CreateCallCount <= 3 {
+		t.Fatalf("Expect 5 completion, get %d", ta.CreateCallCount)
```

Comment:
The number expected in the comment (5) does not match the number in the code (3). Can these be made to match?